### DISCUSSION TITLE
Pydantic 1.9.0 compatibility

### DISCUSSION THREAD

**Discussion by aitorres** [explosion/spaCy#10142] [New Features & Project Ideas]
Hello! I would like to know if it'd be possible for `spacy` to include `pydantic 1.9.0` as a compatible version on the `requirements.txt` file. 

I'm sorry if I opened up this post in the wrong category, but I wasn't 100% sure about where it'd fit better.

**Comment by adrianeboyd** [comment]
It is on our to-do list, but it looks like it's going to require coordinated updates to both `thinc` and `spacy`.

**Comment by dhirschfeld** [comment]
Looks like references to discussions don't create a link here so I'll add it manually - https://github.com/explosion/spaCy/issues/10203

**Comment by svlandeg** [comment]
As an update, support for Pydantic >=1.9 has been included since Thinc 8.1 and spaCy 3.4!
